nodeJS/orms/prisma_orm.md

```diff
@@ -0,0 +1,157 @@
+### Introduction
+
+Tired of writing raw SQL queries in your projects? In this lesson, we'll talk about Object Relational Mappers (ORM for short). ORMs are tools that let you manipulate data in your database and are widely used in the software industry. We'll dive deep into one ORM popular in the Node.js landscape: [Prisma ORM](https://www.prisma.io/orm).
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What ORMs are and why they are used.
+- Introduce Prisma ORM.
+- Describe the features of Prisma ORM.
+
+### Challenges with raw SQLin'
+
+With the couple of projects you've done earlier, you might've run into the tediousness of raw SQL queries. Let's briefly discuss some of these challenges.
+
+#### So much more code
+
+You need a `SELECT` statement? Cool, write a query. Need a `SELECT` statement for a different table? Write another query. Or if your programmer senses are tingling, you could create a utility function for `SELECT`. But now you need a `SELECT` statement for specific columns? You'll have to modify the utility function. What about filters and sorting? You get the point.
```

Comment:
Nit: sorry I missed this one earlier
```suggestion
Do you need a `SELECT` statement? Cool, write a query. Need a `SELECT` statement for a different table? Write another query. Or if your programmer senses are tingling, you could create a utility function for `SELECT`. But now you need a `SELECT` statement for specific columns? You'll have to modify the utility function. What about filters and sorting? You get the point.
```

Reply:
I feel "You need a `SELECT` statement?" reads better than creating a question-ish tone. I'll reserve this change unless we get anyone else's objection.